### ISSUE TITLE
No ability to book ahead

### ISSUE THREAD

**Issue by xeruf** [SergeyMosin/Appointments#326]
I found various issues mentioning the possibility to book ahead (e.g. https://github.com/SergeyMosin/Appointments/issues/291), but using a Weekly Template I am always only offered slots in the current week.

**Comment by xeruf** [comment]
Ah, thank you, how did I miss that! Yes it should really be mentioned in the documentation somewhere, I did a fulltext-search on that, too...

**Comment by colans** [comment]
> Ah, thank you, how did I miss that! Yes it should really be mentioned in the documentation somewhere, I did a fulltext-search on that, too...

Agreed.  The option's not easily discoverable.  I had to search through the issue queue to find this.

Maybe reopen this to add some docs, or in a new issue, @SergeyMosin ?
